src/mcfret.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _MCFRET_ /* ignore */
```

Comment:
It is bad practice to ignore warnings like this ;P

Reply:
Thanks for teaching bad practices!